app/helpers/finance_helper.rb

```diff
@@ -20,4 +34,17 @@ def total_payment(breakdown)
   def total_vat(breakdown, lead_provider)
     total_payment(breakdown) * (lead_provider.vat_chargeable ? 0.2 : 0.0)
   end
+
+  def payment_period
+    index = MILESTONE_DATES.each_index.detect { |date| Time.zone.today.before?(Date.parse(MILESTONE_DATES[date])) }
+    MILESTONE_DATES.slice(index - 1, 2)
+  end
+
+  def pretty_payment_period
+    payment_period.map { |date| Date.parse(date).strftime("%-d %B %Y") }.join(" - ")
+  end
+
+  def cutoff_date
+    Date.parse(payment_period.length < 2 ? MILESTONE_DATES[0] : payment_period[1]).strftime("%-d %B %Y")
+  end
```

Comment:
When is `payment_period` less than 2 elements?

Reply:
it's basically an edge case for the date before the first milestone